types/use-sync-external-store/index.d.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+
+type VoidFunc = () => void
+
+declare module 'use-sync-external-store' {
```

Comment:
[`@types/use-sync-external-store`](https://www.npmjs.com/package/@types/use-sync-external-store) is now available.